app/Providers/AppServiceProvider.php

```diff
@@ -151,15 +126,21 @@ protected function configureAuraDI()
             return config('ratelimiter');
         });
 
-        // Private deployment config settings
-        // @todo move to repo
-        $di->set('site.private', function () use ($di) {
-            $site = $di->get('site.config');
-            $features = $di->get('features');
-            return $site['private']
-                and $features['private']['enabled'];
+        // Multisite db
+        // Move multisite enabled check to class and move to src/App
```

Comment:
when we talk about multisite install, is that the way we have the setup for ushahidi.io clients?

Reply:
I think that's correct, yes